mockwebserver/src/main/kotlin/mockwebserver3/MockWebServer.kt

```diff
@@ -899,6 +899,7 @@ public class MockWebServer : Closeable {
     if (policy.throttlePeriodNanos > 0L) {
       result =
         ThrottledSink(
+          socket = socket,
```

Comment:
Not for this review, but should we be aiming for all operations like this to be on okio socket?

Reply:
Maybe yeah!